dotcom-rendering/src/components/Discussion/CommentForm.stories.tsx

```diff
@@ -65,6 +65,8 @@ export const Default = () => (
 		shortUrl={shortUrl}
 		user={aUser}
 		onAddComment={(comment) => {}}
+		isActive={false}
+		setIsActive={() => {}}
```

Comment:
Stubbing this method will result in story loosing interactivity even though the functionality remains. We think stories should test only the presentational level. I've created a [ticket](https://github.com/guardian/dotcom-rendering/issues/10316) to track some integration tests we may want to add next to increase the test coverage for discussion.

### Before

https://github.com/guardian/dotcom-rendering/assets/19683595/56a2149f-85a8-4e7a-b7d3-b6973a5246b8

### After
https://github.com/guardian/dotcom-rendering/assets/19683595/92dfddba-5896-4022-9b29-e2c9e833f44c

Reply:
Re-introducing interactivity in: 
* https://github.com/guardian/dotcom-rendering/pull/10342